dkc/core/templates/staff/trees_by_file_count.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
```

Comment:
We should be able to use the base from `django-girder-style`. I'll take a look and verify that.